Makefile

```diff
@@ -7,17 +7,18 @@ export $(shell sed 's/=.*//' ./.env)
 export NODE_ENV ?= development
 
 
-.PHONY: build-local api-local web-local print-env run-local
+.PHONY: build api-local web-local print-env run-local
 
-run-local: | build-local api-local web-local
+run-local: | build api-local web-local
```

Comment:
Do we need to build before running local?

Reply:
here build step ensure all build artefacts are upto date.  This is just building each package locally.

Reply:
I think we can save considerable time by only building /common here

Reply:
there could be more than one common packages. How to ensure build them before building/running api and web?

Reply:
Possibly another script? `build-dependencies` that only builds required packages

Reply:
we should look into `watch` options that can look at files changing and run `tsc` for us, this way, frequent changes on any packages can be used in the other.

Reply:
Added watch for common folder and keep watch mode for api and web as built-in script for packages.

Reply:
tsc --watch is working fine, so change this to tsc --watch